docker-compose.yml

```diff
@@ -143,8 +143,9 @@ services:
     container_name: blip
     # volumes:
     #   - $TP_BLIP_DIR:/app:cached
-    #   - $TP_VIZ_DIR:/viz:cached
+    #   - $TP_VIZ_DIR:/@tidepool/viz:cached
```

Comment:
What does the `/@tidepool` do?

Reply:
Because the `viz` (and all future) npm module is 'namespaced' under tidepool, it's package name, and resulting path in the `node_modules` directory are prefixed with the `@`

To simplify the `tidepool` bash script usage, it was better if it was mounted at a path matching it's package name.